grammars/javascript.cson

```diff
@@ -702,11 +702,11 @@
   {
     # console
     # console.log(arg1, "arg2", [...])
-    'begin': '\\bconsole\\b'
+    'begin': '\\bconsole\\b(?!\\$)'
```

Comment:
Maybe we can do something like -

`'begin': '(?<!\\$)\\bconsole\\b(?!\\$|//|/\*)'`